docs/tutorials/02_neural_network_classifier_and_regressor.ipynb

```diff
@@ -46,7 +46,9 @@
     "\n",
     "from typing import Union\n",
     "\n",
-    "from qiskit_machine_learning.exceptions import QiskitMachineLearningError"
+    "from qiskit_machine_learning.exceptions import QiskitMachineLearningError\n",
+    "\n",
+    "np.random.seed(42)"
```

Comment:
Can you please replace `np.random.*` with `algorithm_globals.random.*` ? Same as in the `01` tutorial.